
Testing
  0%|                                                                                           | 0/2000 [00:00<?, ?it/s]/home/dogoulis/anaconda3/lib/python3.8/site-packages/torch/nn/functional.py:718: UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at  /opt/conda/conda-bld/pytorch_1623448278899/work/c10/core/TensorImpl.h:1156.)
  return torch.max_pool2d(input, kernel_size, stride, padding, dilation, ceil_mode)














































 99%|███████████████████████████████████████████████████████████████████████████████▌| 1989/2000 [01:33<00:00, 22.46it/s]
Test loss is 0.001125105243879716
[[1000    0]
 [   0 1000]]
      0     1
0  1000     0
100%|████████████████████████████████████████████████████████████████████████████████| 2000/2000 [01:34<00:00, 21.22it/s]
Traceback (most recent call last):
  File "evaluation.py", line 231, in <module>
    main()
  File "evaluation.py", line 225, in main
    log_conf_matrix(y_true=y_true, y_pred=y_pred)
  File "evaluation.py", line 153, in log_conf_matrix
    cf_matrix = wandb.Table(cf_matrix_pd)
  File "/home/dogoulis/anaconda3/lib/python3.8/site-packages/wandb/data_types.py", line 253, in __init__
    self._init_from_list([], columns, optional, dtype)
  File "/home/dogoulis/anaconda3/lib/python3.8/site-packages/wandb/data_types.py", line 268, in _init_from_list
    self._assert_valid_columns(columns)
  File "/home/dogoulis/anaconda3/lib/python3.8/site-packages/wandb/data_types.py", line 260, in _assert_valid_columns
    assert type(columns) is list, "columns argument expects a `list` object"
AssertionError: columns argument expects a `list` object